docs/internals/contributing/writing-documentation.txt

```diff
@@ -179,10 +184,21 @@ Link check
 
 Links in documentation can become broken or changed such that they are no
 longer the canonical link. Sphinx provides a builder that can check whether the
-links in the documentation are working. From the ``docs`` directory, run ``make
-linkcheck``. Output is printed to the terminal, but can also be found in
+links in the documentation are working. From the ``docs`` directory, run:
+
+.. console::
+
+     $ make linkcheck
+
+Output is printed to the terminal, but can also be found in
 ``_build/linkcheck/output.txt`` and ``_build/linkcheck/output.json``.
 
+.. admonition:: Be careful with the time
```

Comment:
I like having that note but the heading isn’t super clear to me.

Reply:
Do you have suggestions for a clearer title?

Reply:
This can be a warning without a title

Reply:
Make sense to me. I removed the title, and updated the commit.

Reply:
> This can be a warning without a title

@sarahboyce I just find out that the `admonition` directive require a title :)

```
...writing-documentation.txt:196: ERROR: Error in "admonition" directive:
1 argument(s) required, 0 supplied.
```

So I added a tittle again in the code because the use of the `note` directive is discouraged in the same documents.

> To improve readability, use ``.. admonition:: Descriptive title`` rather tha  ``.. note::``

See https://docs.djangoproject.com/en/stable/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

> I like having that note but the heading isn’t super clear to me.

@thibaudcolas I tried to add a clearer title

Reply:
I think warnings can have no title

Reply:
> I think warnings can have no title (`.. warning::`)

Using `.. warning::` seems a good idea, thanks, I checked that it's widely used in the code. I just updated the PR. Thanks.